src/main/java/com/minecolonies/coremod/client/gui/WindowCitizen.java

```diff
@@ -348,6 +540,38 @@ private void createSkillContent()
           LanguageHandler.format("com.minecolonies.coremod.gui.citizen.skills.dexterity", citizen.getDexterity()));
     }
 
+    private ImmutableList<IRequest> getOpenRequestsOfCitizen()
+    {
+        ArrayList<IRequest> requests = new ArrayList<>();
+        if (citizen.getWorkBuilding() != null)
+        {
+            requests.addAll(getOpenRequestsOfCitizenFromBuilding(citizen.getWorkBuilding()));
+        }
+
+        if (citizen.getHomeBuilding() != null)
+        {
+            requests.addAll(getOpenRequestsOfCitizenFromBuilding(citizen.getHomeBuilding()));
+        }
+
+        final BlockPos playerPos = Minecraft.getMinecraft().player.getPosition();
+        requests.sort(Comparator.comparing((IRequest request) -> request.getRequester().getDeliveryLocation().getInDimensionLocation().getDistance(playerPos.getX(), playerPos.getY(), playerPos.getZ())).thenComparingInt(Object::hashCode));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 238 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00103)